src/reactions/github/reaction.ts

```diff
@@ -17,7 +18,11 @@ export interface ReactionStatus {
 }
 
 export abstract class Reaction<P = WebhookPayload> {
-	public constructor(private twitchChat: TwitchChat, private streamlabs: StreamLabs) {}
+	public constructor(
+		private twitchChat: TwitchChat,
+		private streamlabs: StreamLabs,
+		protected config: Config = getConfig(),
```

Comment:
We already receive the configuration in the `reactionBuild` method, we don't need that default value.